aio/deploy/helm-chart/kubernetes-dashboard/Chart.yaml

```diff
@@ -14,7 +14,7 @@
 
 apiVersion: v1
 name: kubernetes-dashboard
-version: 2.4.0
+version: 2.4.1
```

Comment:
Could you bump minor version instead of patch? According to semver this is a new feature.

Reply:
Done. changed it to 2.5.0